form.blade.php

```diff
@@ -0,0 +1,204 @@
+@extends('layouts.main')
+@section('form')
+<div class="accordion container" id="accordionExample">
+    <h3>Education Details</h3>
+    {{-- 10th accordion --}}
+    <div class="">
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingone_ssc">
+        <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
```

Comment:
change `#collapseOne ` to `#ssc_details` or something meaningful

Reply:
i have changed collapseOne to #ssc_details